lib/intelligent_foods/resources/order.rb

```diff
@@ -12,24 +10,55 @@ class Order < IntelligentFoods::Object
     PROCESSED = "processed"
 
     def initialize(args = {})
-      @status = INITIALIZED
       super
     end
 
+    def self.build_from_response(data)
+      order = build(data)
+      order[:items] = OrderItem.build(data[:items])
+      order[:ship_to] = Recipient.build(data[:ship_to])
+      order
+    end
+
+    def create!
+      uri = URI("#{IntelligentFoods.base_url}/order")
+      request = Net::HTTP::Post.new(uri)
+      request.body = request_body.to_json
+      request["content-type"] = "application/json"
```

Comment:
how many places are these three lines replicated now? is it time to think about pushing this into the Client class?